R/boxr-package.R

```diff
@@ -1,22 +1,18 @@
 #' boxr: access the Box API
 #' 
-#' A light-weight, high-level R interface for the box.com API
+#' A lightweight, high-level R interface for the [box.com](https://box.com) API
```

Comment:
The docs reference the Box.com API a lot, but always link to normal box.com should these link to https://developer.box.com/docs, or something more API specific? More of a thought than a change request...